docs/pages/installation/windows.md

```diff
@@ -0,0 +1,48 @@
+### Windows
+
```

Comment:
The prerequisites need to be mentioned. Docker is one of them. As far as I'm aware, one also needs a Linux VM, to easily work with Docker on Windows.